components/automate-cli/pkg/verifyserver/models/batchcheck.go

```diff
@@ -7,10 +7,17 @@ import (
 	"strings"
 
 	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/sirupsen/logrus"
+
 	"github.com/chef/automate/lib/config"
 	"github.com/gofiber/fiber/v2"
 )
 
+const (
+	AWS_S3            = "s3"
+	GCP_CLOUD_STORAGE = "gcp"
```

Comment:
Are we using GCP or GCS, please decide on one. In the previous PR, the constant was GCS, and in this its GCP. As per standalone it should be GCS